pypesto/hierarchical/spline_approximation/problem.py

```diff
@@ -249,6 +314,58 @@ def spline_inner_parameters_from_measurement_df(
     return inner_parameters
 
 
+def noise_inner_parameters_from_parameter_df(
+    petab_problem: 'petab.Problem',
+    amici_model: 'amici.Model',
+) -> List[SplineInnerParameter]:
+    """Create list of inner free noise parameters from PEtab parameter table."""
+    # Select the nonlinear monotone measurements.
+    measurement_df = petab_problem.measurement_df
+    measurement_df = measurement_df[
+        measurement_df[MEASUREMENT_TYPE] == NONLINEAR_MONOTONE
+    ]
+
+    observable_ids = amici_model.getObservableIds()
+
+    # Create a dictionary with unique pairs of observable id
+    # and noise parameter from the measurement table.
+    noise_parameter_to_observable = {}
+    for _, row in measurement_df.iterrows():
+        observable_id = row[OBSERVABLE_ID]
+        noise_parameter_id = row[NOISE_PARAMETERS]
+        noise_parameter_to_observable[noise_parameter_id] = observable_id
+
+    noise_parameters = []
+
+    parameter_df = petab_problem.parameter_df.reset_index()
+    # Create noise inner parameters.
+    for _, row in parameter_df.iterrows():
+        if row[PARAMETER_ID] not in noise_parameter_to_observable:
+            continue
+        if petab.is_empty(row.get(PARAMETER_TYPE)) or not row[ESTIMATE]:
+            continue
+        else:
+            estimate = True
+        observable_id = noise_parameter_to_observable[row[PARAMETER_ID]]
+        group = observable_ids.index(observable_id) + 1
+
+        noise_parameters.append(
+            SplineInnerParameter(
+                inner_parameter_id=row[PARAMETER_ID],
+                inner_parameter_type=InnerParameterType.SIGMA,
+                scale=LIN,
+                lb=row[LOWER_BOUND],
+                ub=row[UPPER_BOUND],
+                observable_id=observable_id,
+                group=group,
+                index=None,
+                estimate=estimate,
+            )
+        )
```

Comment:
Can simply set `estimate=True`

```suggestion
    # Create noise inner parameters.
    for _, row in parameter_df.iterrows():
        if row[PARAMETER_ID] not in noise_parameter_to_observable:
            continue
        if petab.is_empty(row.get(PARAMETER_TYPE)) or not row[ESTIMATE]:
            continue
        observable_id = noise_parameter_to_observable[row[PARAMETER_ID]]
        group = observable_ids.index(observable_id) + 1

        noise_parameters.append(
            SplineInnerParameter(
                inner_parameter_id=row[PARAMETER_ID],
                inner_parameter_type=InnerParameterType.SIGMA,
                scale=LIN,
                lb=row[LOWER_BOUND],
                ub=row[UPPER_BOUND],
                observable_id=observable_id,
                group=group,
                index=None,
                estimate=True,
            )
        )
```